rocketchat/values.yaml

```diff
@@ -334,3 +334,6 @@ microservices:
 nats:
   nats:
     image: nats:2.4-alpine
+
+federation:
+  enabled: false
```

Comment:
should we have a toggle to specify synapse as the home server?  That way later we can let people specify dendrite for their deployments.